templates/components/amp/products/list-item.html

```diff
@@ -1,6 +1,6 @@
 <article class="listItem">
     <figure class="listItem-figure">
-        <amp-img src="{{getImage image 'productgallery_size' (cdn theme_settings.default_image_product)}}" width="{{first (split theme_settings.productgallery_size 'x')}}" height="{{last (split theme_settings.productgallery_size 'x')}}" layout="responsive" alt="{{image.alt}}"></amp-img>
+        <amp-img src="{{getImageSrcset image (cdn theme_settings.default_image_product) use_default_sizes=true}}" layout="responsive" alt="{{image.alt}}"></amp-img>
```

Comment:
`srcset` is polyfilled by AMP, so there's no need to provide a fallback `src` for amp-img, you can just use `srcset`.

Reply:
This is inconsistent with the usage in `templates/components/amp/products/card.html`. That uses the attribute `srcset` where this one uses `src`. Which is correct?

Reply:
This file was a miss, I've fixed it.